src/main/java/com/google/devtools/build/lib/bazel/bzlmod/SingleExtensionEvalFunction.java

```diff
@@ -347,6 +401,32 @@ private void validateAllImportsAreGenerated(
     }
   }
 
+  /**
+   * @return usages with all information removed that does not influence the evaluation of the
```

Comment:
Google style usually suggests that "Returns X" instead of "@return X" if there's only a `@return` clause